doc/media-info-enhancements/media-info.md

```diff
@@ -0,0 +1,170 @@
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Type of directly-attached cable</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>string, one of "DAC", "RJ45", "FIBER", "AOC", "ACC"</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>Layer-1 configuration lookup, e.g NPU interface type</td>
+	</tr>
+</table>
+<h4>display_name</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>String suitable for displaying description of transceiver</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>string</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>CLI show command(s)</td>
+	</tr>
+	<tr>
+		<td valign="top">Notes</td>
+		<td>Rather than the CLI code attempt to construct descriptive string for an installed transceiver, from the information in the TRANSCEIVER_TABLE, xcvrd can construct a short descriptive string for use by CLI show commands.  Since transceiver "knowledge" is inside xcvrd, it is reasonable for it to compose the displayable descriptive string, rather than the CLI.</td>
+	</tr>
+	<tr>
+		<td valign="top">Examples</td>
+		<td>"SFP+ 10GBASE-DWDM-TUNABLE", "QSFP28 100GBASE-CR4-0.5M"</td>
+	</tr>
+</table>
+<h4>lane_count</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Number of lanes</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>integer, 1..N</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>Validation of port breakout configurations, i.e. whether or not transceiver supports breakout</td>
+	</tr>
+</table>
+<h4>media_interface</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Media interface</td>
+	</tr>
+	<tr>
+		<td valign="top">Value</td>
+		<td>string, one of "SR", "CR", "ZR", "DR", "EDR", "BASE-T"</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>Layer-1 configuration lookup, e.g preemphasis</td>
+	</tr>
+</table>
+<h4>form_factor</h4>
+<table style="border-style:none;">
+	<tr>
+		<td valign="top">Description</td>
+		<td>Detailed transceiver type, i.e. form factor</td>
+	</tr>
+	<tr>
+		<td valign="top">Value(s)</td>
+		<td>string, one of "SFP", "SFP+", "SFP28", "QSFP+", "QSFP28", "QSFP28-DD", "QSFP56-DD", "RJ45"</td>
+	</tr>
+	<tr>
+		<td valign="top">Needed by</td>
+		<td>Layer-1 configuration lookup, e.g preemphasis</td>
+	</tr>
+	<tr>
+		<td valign="top">Notes</td>
+		<td>This field is intended as a replacement for the existing "type_abbrv_name" field.  That field does not fully discriminate a transceiver "type".  For example, a QSFP-DD transceiver may be a QSFP28-DD or QSFP56-DD.  Also, other types, such as "SFP28", are missing.<br/>Rather than introduce new values, such as "SFP28", "QSFP-DD28" and "QSFP-DD56" for the existing field, and deprecating old values, such as "QSFP-DD", a new field will be introduced, that fully discriminates a "type", and the old field shall be deprecated.</td>
```

Comment:
i do not understand rationale to introduce a new field here. You can deprecate the QSFP-DD value, and introduce new values such as QSFP-DD28 and QSFP-DD56. why do you need to deprecate the whole field?

Reply:
It is purely a judgement call.  The main goal is to maintain backward compatibility.  Whether backward compatibility is done by deprecating an existing field and introducing a new field, or by deprecating values and introducing  new values in an existing field -- both accomplish the same.  It was felt that deprecating a field makes it more "obvious"...